src/protocolgame.cpp

```diff
@@ -837,6 +847,134 @@ void ProtocolGame::parseLookInBattleList(NetworkMessage& msg)
 	addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerLookInBattleList, player->getID(), creatureId);
 }
 
+void ProtocolGame::parseExecuteCommand(const std::string& text) {
+	size_t pos = text.find(' ');
+	std::string command = text.substr(1, pos - 1);
+	if (command.empty()) {
+		return;
+	}
+
+	if(command == "commands") {
+		player->sendChannelMessage("", "Available commands:", TALKTYPE_CHANNEL_O, CHANNEL_CAST, false);
```

Comment:
Thank you for pointing this out, do you have any tip on how i SHOULD handle these things to not data race? Thank you.